packages/frontend/pages/strategies.tsx

```diff
@@ -352,25 +355,54 @@ const Strategies: React.FC = () => {
                       tooltip={Tooltips.StrategyProfitThreshold}
                     />
                   ) : (
-                    <StrategyInfoItem
-                      value={
-                        '~' +
-                        lowerPriceBandForProfitability.toFixed(2) +
-                        ' - ' +
-                        upperPriceBandForProfitability.toFixed(2)
-                      }
-                      label={
-                        'Approx Profitable (' + (profitableMovePercentV2 * 100).toFixed(2) + '%) between ETH Prices ($)'
-                      }
-                      tooltip={Tooltips.StrategyProfitThreshold}
-                    />
+                    
+                    crabLoading ? (
+                      <>
+                          <StrategyInfoItem
+                          value='-'
+                          label={
+                            'Approx Profitable between ETH Prices ($)'
+                          }
+                          tooltip={Tooltips.StrategyProfitThreshold}
+                          />
+
+                          <StrategyInfoItem
+                          link="https://squeeth.opyn.co/vault/286"
+                          value='-'
+                          label="Collat Ratio (%)"
+                          tooltip={Tooltips.StrategyCollRatio}
+                          />
+                      </>
+                    ) : (
+                    <>
+                      <StrategyInfoItem
+                        value={
+                          '~' +
+                          lowerPriceBandForProfitability.toFixed(2) +
+                          ' - ' +
+                          upperPriceBandForProfitability.toFixed(2)
+                        }
+                        label={
+                          'Approx Profitable (' + (profitableMovePercentV2 * 100).toFixed(2) + '%) between ETH Prices ($)'
+                        }
+                        tooltip={Tooltips.StrategyProfitThreshold}
+                      />
+
+                      <StrategyInfoItem
+                      link="https://squeeth.opyn.co/vault/286"
+                      value={collatRatio === Infinity ? '0.00' : collatRatio.toString()}
```

Comment:
Instead of 0.0 can we make it - ?
Is it ever Infinity?

Reply:
Updated to read '-'
Its only Infinity when the contract information is not loaded.